pyproject.toml

```diff
@@ -107,9 +109,10 @@ ignore = [
 
 [tool.ruff.lint.per-file-ignores]
 "tests/*" = [
-    "D",    # pydocstyle: no docstrings required for tests
-    "E501", # line-too-long: embedded test data in "fmt: off" blocks is ok
-    "S",    # bandit: Not running bandit on tests
+    "D",      # pydocstyle: no docstrings required for tests
+    "E501",   # line-too-long: embedded test data in "fmt: off" blocks is ok
+    "S",      # bandit: Not running bandit on tests
+    "RUF012", # ruff: mutable-class-default
```

Comment:
I guess we have a bunch of these? Could mention that in the comment but no need to modify just for that